src/CustomStyleCommand.js

```diff
@@ -63,10 +63,12 @@ function markApplies(doc, ranges, type) {
         return false;
       }
       can = node.inlineContent && node.type.allowsMarkType(type);
+      return true;
```

Comment:
What should this function be returning?  Currently it returns 3 possible values.    false, true, and the value of tr that was passed in as a parameter.

The code where it is being called (same file, line 163) **always** assumes the response will be a transaction.

Reply:
Yes the toggleCustomStyle function returns a trasaction object, fixed issue with inconsistent return value.